selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/DefaultAndroidEmulator.java

```diff
@@ -143,14 +145,26 @@ public String getAvdName() {
 
   public static List<AndroidEmulator> listAvailableAvds() throws AndroidDeviceException {
     List<AndroidEmulator> avds = Lists.newArrayList();
+    String avdParam = "avds";
+    File clu = null;
+
+    if(AndroidSdk.getAndroidVersionNumber() >= SDK_VERSION_FOR_AVD_MANAGER ){
+      clu =  AndroidSdk.avdManager();
+      avdParam = "avd";
+    }
+    else {
```

Comment:
Minor nitpick: Move the else to the previous line along with the closing bracket